tests/target/issue-2321/trailing_comma_always.rs

```diff
@@ -0,0 +1,9 @@
+// rustfmt-brace_style: PreferSameLine
+// rustfmt-trailing_comma: Always
+
+fn lorem<S, T,>(lorem: S, ipsum: T,)
+where
+    S: Add + Sub,
+    T: Mul + Div, {
+    // body
+}
```

Comment:
Can we also include a where clause with with a single bound. I like that you've included tests with `trailing_comma: Always` and `trailing_comma: Vertical`. Could you also expand the tests to include the other combinations of `brace_style`

Reply:
Thank you for the review. I added test case for brace_style AlwaysNextLine and SameLineWhere pattern.

Reply:
Thank you for including those additional test cases! I'll have some time later this week to give this a more thorough review 😁